### ISSUE TITLE
[Grid] Page is not responsive after switching between tabs in Chrome. 

### ISSUE THREAD

**Issue by viacheslav897** [telerik/kendo-angular#3037]
**Describe the bug**
The page is not responsive few seconds after switching between tabs in Chrome

**To Reproduce**
1. Open a page with a grid component which contains more than 10 records in Google Chrome
2. Switch to another Google Chrome window (Alt + Tab)
3. Switch back to the page with a grid.

**Browser**
 - OS: Windows 10
 - Browser: Chrome
 - Version: kendo-angular-grid: 4.7.2

**Comment by dimitar-pechev** [comment]
Cannot reproduce the described behavior with the site demos ([example ref](https://www.telerik.com/kendo-angular-ui/components/grid/how-to/filter-all-columns/)). Could you share a StackBlitz example, where the problem is reproduced?

Also, please make sure that the problem is not caused by a third-party extension in Chrome. You can test to see it that's the case by running your application in an incognito window.

**Comment by ChristianBjerre** [comment]
Thanks for your feedback @dimitar-pechev.

1. The problem does not appear in Firefox.
2. The problem also appears when we run our application in an incognito window.

**Comment by dimitar-pechev** [comment]
@ChristianBjerre Could you share a runnable example where the issue is reproduced?

**Comment by ChristianBjerre** [comment]
Yes but only in our application. Let me give you access to the solution and show you how it appears.
